### PR TITLE
Use requests instead of sse cli commands for token login

### PR DESCRIPTION
**Pull request by miklosduma**
In some cases the token tests were failing due to a known argparse library bug: https://bugs.python.org/issue9334

The token secret generated by SSE sometimes starts with a dash, which argparse mistakes for another argument. Since this is a rare edge case and we don't presently use the login command with tokens, and the affected tests are black-box tests for SSE core, I only propose a fix for the tests, not the behaviour itself.

Example of failure:
https://travis-ci.com/github/opensparkl/sse_cli/jobs/379091802

### CODE REVIEW

File: docs/ownership/register_by_csr/register_by_csr.rst [hunk 1]
```diff
@@ -262,15 +262,24 @@ Obtain a token and use it.
...
-    >>> sparkl('login', user, token_secret, token=token_name)
-    {...'administrator': True,...'name': 'owner@localhost'...'tag': 'user'}
-    >>> sparkl('user')
+    >>> token_login_response = requests.post(
+    ...     SSE_URL_HTTPS + "/sse_cfg/user",
+    ...     verify=False,
+    ...     headers={"Accept": "application/json"},
+    ...     params={"email": user,
+    ...           "password": token_secret,
+    ...           "token": token_name})
```
Comment (lucafavatellawork) [inline docs/ownership/register_by_csr/register_by_csr.rst:271]: Strange indentation.

File: docs/ownership/claim_cert_by_csr/claim_cert_by_csr.rst [hunk 2]
```diff
@@ -326,15 +326,24 @@ Obtain a token and use it.
...
-    >>> sparkl('login', user, token_secret, token=token_name)
-    {...'administrator': True,...'name': 'owner@localhost'...'tag': 'user'}
-    >>> sparkl('user')
+    >>> token_login_response = requests.post(
+    ...     SSE_URL_HTTPS + "/sse_cfg/user",
+    ...     verify=False,
+    ...     headers={"Accept": "application/json"},
+    ...     params={"email": user,
+    ...           "password": token_secret,
+    ...           "token": token_name})
```
Comment (lucafavatellawork) [inline docs/ownership/claim_cert_by_csr/claim_cert_by_csr.rst:335]: Strange indentation.

File: docs/ownership/claim_cert_and_privkey/claim_cert_and_privkey.rst [hunk 3]
```diff
@@ -191,15 +191,24 @@ Obtain a token and use it.
...
-    >>> sparkl('login', user, token_secret, token=token_name)
-    {...'administrator': True,...'name': 'owner@localhost'...'tag': 'user'}
-    >>> sparkl('user')
+    >>> token_login_response = requests.post(
+    ...     SSE_URL_HTTPS + "/sse_cfg/user",
+    ...     verify=False,
+    ...     headers={"Accept": "application/json"},
+    ...     params={"email": user,
+    ...           "password": token_secret,
+    ...           "token": token_name})
```
Comment (lucafavatellawork) [inline docs/ownership/claim_cert_and_privkey/claim_cert_and_privkey.rst:200]: Strange indentation.

**Review by jacobythwaites — APPROVED** [review]

**Review by lucafavatellawork — APPROVED** [review]
